app/packs/controllers/auto_citation_controller.js

```diff
@@ -36,16 +36,24 @@ export default class extends Controller {
   }
 
   get authors() {
-    return this.contributorRoleTargets.map((roleField, index) => {
+    return this.contributorRoleTargets.filter(elem => elem.disabled == false).map((roleField, index) => {
```

Comment:
Should this be a method called `contributorRoles` since it's very similar to `contributorFirsts` an `contributorLasts`?

Reply:
It returns authors, not roles.

Sidenote: I'm changing to factor out the roles into a separate method, to make this a bit clearer.